issue.go

```diff
@@ -96,29 +96,58 @@ type Epic struct {
 	Done    bool   `json:"done" structs:"done"`
 }
 
+type Description struct {
+	Version int       `json:"version,omitempty"`
+	Type    string    `json:"type,omitempty"`
+	Content []Content `json:"content,omitempty"`
+}
+
+type Content struct {
+	Type    string    `json:"type,omitempty"`
+	Text    string    `json:"text,omitempty"`
+	Content []Content `json:"content,omitempty"`
+}
+
 // A description field may be a plain old string or a document structure. This
 // is extremely annoying as they have the same field name and cannot both unmarshal
 // to the same type without jumping through hoops.
-type Document string
+type Document struct {
+	Description Description `json:"description"`
+	Raw         []byte      `json:"raw"`
```

Comment:
I don't think we need to carry around this raw data. (See also the note at `String()`)